source/common/http/conn_manager_utility.cc

```diff
@@ -319,6 +319,11 @@ void ConnectionManagerUtility::mutateResponseHeaders(Http::HeaderMap& response_h
     response_headers.insertRequestId().value(*request_headers.RequestId());
   }
 
+  if (protocol == Protocol::Http2) {
+    response_headers.removeKeepAlive();
```

Comment:
Isn't this a hop by hop header which we should clear for HTTP/1.1 as well?  We clear it for both H2 and H1 on the request path.

I'm having trouble finding anything in the spec about proxy connection but given we clear consistently in the request headers I wonder if it's the same

Reply:
I suspected so, but I didn't have any documentation at hand like I did for h/2, so I only did h/2 to begin with. At this point I'm pretty sure it's the same so I'll remove the protocol check.